slither/printers/summary/printer_human_summary.py

```diff
@@ -0,0 +1,102 @@
+"""
+Module printing summary of the contract
+"""
+import logging
+
+from slither.detectors.detectors import Detectors
+from slither.printers.abstractPrinter import AbstractPrinter
+from slither.utils.code_complexity import compute_cyclomatic_complexity
+from slither.utils.colors import green, red, yellow
+
+
+class PrinterHumanSummary(AbstractPrinter):
+    ARGUMENT = 'print-human-summary'
+    HELP = 'Print an human readable summary of the contracts'
+
+    @staticmethod
+    def get_summary_erc20(contract):
+        txt = ''
+        functions_name = [f.name for f in contract.functions]
+        state_variables = [v.name for v in contract.state_variables]
+
+        if 'paused' in functions_name or 'pause' in functions_name:
+            txt += "\t\t Can be paused? : {}\n".format(yellow('Yes'))
+        else:
+            txt += "\t\t Can be paused? : {}\n".format(green('No'))
+
+        if 'mint' in functions_name:
+            if not 'mintingFinished' in state_variables:
+                txt += "\t\t Minting restriction? : {}\n".format(red('None'))
+            else:
+                txt += "\t\t Minting restriction? : {}\n".format(yellow('Yes'))
+        else:
+            txt += "\t\t Minting restriction? : {}\n".format(green('No Minting'))
+
+        if 'increaseApproval' in functions_name:
+            txt += "\t\t ERC20 race condition mitigation: {}\n".format(green('Yes'))
```

Comment:
Consider being more explicit here as:
* the check itself doesn't necessarily mean that there is race condition mitigation - at least I assume that it depends on how the `increaseApproval` is implemented, right?
* someone needs to look into the code to understand what does it mean
* maybe we should change `Yes` and `No` here into: `has increaseApproval function`, `misses increaseApproval function`?